src/qdrant_client/builers/vectors_config.rs

```diff
@@ -0,0 +1,51 @@
+use crate::qdrant::vectors_config::Config;
+use crate::qdrant::{VectorParams, VectorParamsMap, VectorsConfig};
+use std::collections::HashMap;
+
+const DEFAULT_VECTOR_NAME: &str = "";
+
+#[derive(Debug, Clone, Default)]
+pub struct VectorsConfigBuilder {
+    params: HashMap<String, VectorParams>,
+}
+
+impl VectorsConfigBuilder {
+    pub fn add_named_vector_params(
+        &mut self,
+        name: impl Into<String>,
+        params: impl Into<VectorParams>,
+    ) -> &mut Self {
+        self.params.insert(name.into(), params.into());
+        self
+    }
+
+    pub fn add_vector_params(&mut self, params: impl Into<VectorParams>) -> &mut Self {
```

Comment:
Not sure if `add` is the right suffix here. It sounds to me as if you can call it multiple times. What do you think about naming it `with_vector_params` or just `vector_params` as the other setter functions?

Reply:
then it won't be consistent with `add_named_vector_params` (which is not setter), because it won't overwrite data there with non-default name

Reply:
I'm not sure whether it needs to be treated that similar to `add_named_vector_params` because one is basically a setter and one not. 
It was just a suggestion because I can imagine user getting confused by this. With the comment I think it'd be fine either way so feel free to mark this as resolved